test/providers/dynect_test.rb

```diff
@@ -6,6 +6,10 @@ def setup
     @dyn = Provider::DynECT.new(zone: @zone_name)
   end
 
+  def test_supports_alias_by_default
+    refute Provider::DynECT.supports_alias?, 'DynECT should NOT support ALIAS by default'
```

Comment:
No predicate here though, `Provider::DynECT.supports_alias?` is a Boolean, same with the previous comment.